src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, Color color) {
+        setFigure(FigureType.CIRCLE);
+        setRadius(radius);
+        setColor(color);
+        double area = Math.round(getCalculatedArea());
+        setArea(area);
+    }
+
+    public Circle() {
+        setFigure(FigureType.CIRCLE);
+        setRadius(2);
+        setColor(Color.BLUE);
+        double area = Math.round(getCalculatedArea());
+        setArea(area);
+    }
+
+    @Override
+    public double getCalculatedArea() {
+        return Math.PI * radius * 2;
```

Comment:
The formula for calculating the area of a circle is incorrect. It should be `Math.PI * radius * radius` instead of `Math.PI * radius * 2`. Please correct this to ensure the area is calculated accurately.